templates/predict.ipynb

```diff
@@ -21,34 +21,27 @@
    ],
```

Comment:
<p>Here we should spell out again what the default assumption is: that all segmentation TIFs are in a single directory with a specified suffix. Then if people want to change things with segmentation_naming_convention, we can tell them to copy it in and modify. But it'll make the notebook cleaner if we default to not having that in there. </p><p> </p><p>Also, I think <code>ark</code> got rid of feature_0 and uses a more intuitive name now, you can double check.  </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/angelolab/cell_classification/pull/65/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='5'/>